fares-data-build-tool/create-data
# feat: CFD-378 Banner to return from global settings

# Description

-   Allow the returning from global setting back to where the user came from
NOTE: This only currently works on the home page redirect because the other points in the journey don't exist

# Testing instructions

-   Go to home page and click the operator settings link
-   Check the banner displays that indicates you can navigate back
-   Check that the banner is present on both the time restrictions and passenger type pages
-   On any of the three pages click the button and check you navigate back to the home page
-   Hit the browser back button and check that the banner no longer displays

# Type of change
